stage/all-services.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# Generate a random SHA-512 hash
+generate_random_sha512() {
+  head -c 64 /dev/urandom | sha512sum | awk '{print $1}'
+}
+
+# Export secrets as environment variables
+export_secrets() {
+  export ENV_TAG="staging"
+  export MARIADB_ROOT_PASSWORD=$(generate_random_sha512)
+  export MARIADB_PASSWORD=$(generate_random_sha512)
+  export DB_PASSWORD_WIKI=$MARIADB_PASSWORD
+  export LOGIN_PASSWORD_WIKI=$(generate_random_sha512)
+  export ADMIN_PASSWORD_WIKI=$(generate_random_sha512)
+  export BAN_PASSWORD_WIKI=$(generate_random_sha512)
+  export SITE_URL="staging-wiki.wetfish.net"  # Update with your actual URL
+  export ALLOWED_EMBEDS="/^.*\\.example\\.com$/i"
```

Comment:
Same as above, if this is going to be hardcoded we should use the actual value.

`ALLOWED_EMBEDS=/^.*\.wetfish.net$/i`